reconcile/utils/ocm.py

```diff
@@ -99,14 +97,22 @@ def _init_request_headers(self):
             "accept": "application/json",
         }
 
+    @staticmethod
+    def _ready_for_app_interface(cluster: dict[str, Any]) -> bool:
+        return (
+            cluster["managed"]
+            and cluster["state"] == STATUS_READY
+            and "storage_quota" in cluster
```

Comment:
i'm ok with excluding rosa cluster temporarily to not fail integrations and PR checks. in a follow up PR this needs to be redefined together with some other changes in ocm.py to handle the ROSA differences properly.